### DISCUSSION TITLE
Debugging a frozen module

### DISCUSSION THREAD

**Discussion by davefes** [micropython/micropython#13158] [Using MicroPython] [answered]
I found this clue:

`py: Only search frozen modules when '.frozen' is found in sys.path. `

What if you want to work on one out of several frozen modules and you don’t have enough flash to “unfreeze” all of them?

The problem I am having is trying to debug `micropython/lora` on a STM32WL5x board with very limited space on it’s LFS2 filesystem.

Is there some way to have a file on `/flash` override one that is baked-in?

**Comment by andrewleech** [comment, marked as answer]
If you change sys.path to have flash before frozen you can replace one module, eg
`sys.path.insert(0, '/flash')`

**Reply by davefes** [reply]
Great I will give a try.
